lib/services/resourceManagement/package.json

```diff
@@ -6,7 +6,7 @@
     "Zavery, Amar <amzavery@microsoft.com>",
     "Srinivasan, Vivek <visriniv@microsoft.com>"
   ],
-  "version": "7.3.0",
+  "version": "8.0.0",
```

Comment:
Why would this change need a major version update?

Reply:
Just in case some user may use this reference explicitly

Reply:
Actually I think we should remove the dependency in ms-rest-azure https://github.com/Azure/azure-sdk-for-node/issues/5157 here

Reply:
Though moment is imported in these files, it is not exported and therefore a user would have no visibility to it. So, a major version update is not needed.

Reply:
Okay, Changed. Please review it again.